returntocorp/sgrep-rules
# increasing historical semgrep version

This action:

* runs every week
* increases the semgrep-version in `semgrep-rules-test-historical.yml` by 1
* creates a PR with version increase

This automates the need to increase the semgrep-version manually in `semgrep-rules-test-historical.yml` every week. This makes sure the historical semgrep-version test correlates to ~1 month behind our current semgrep version test.

_To Test:_
Go to `actions`
Run `increase-historical-semgrep-script.yml` manually.
Check that this action makes a PR that increases the version number for `semgrep-rules-test-historical.yml`. 

Example PR this creates: https://github.com/returntocorp/semgrep-rules/pull/2047

NOTE:
This workflow uses a PAT that I've generated. Perhaps it would be better to generate a PAT with a company-wide account instead. @minusworld what do you think?
